### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by mimischi**
The introduction contains a small typo, leading to a line break in the `.tex` file. I'm still reading and will update the PR if I discover anything else.

### CODE REVIEW

**Comment by mrshirts** [comment]
IF you can fix this today @davesmith4398, I think we can still get it into the final issue.

File: membranes_good.tex [hunk 1]
```diff
@@ -1088,7 +1088,7 @@ \subsubsection{Other/hybrid construction methods}
 \label{subsubsec:otherandhybridmtds}
 One fairly simple alternative to building a membrane oneself is downloading a pre-equilibrated membrane from a lipid library ~\cite{Javanainen2016}.
 Membranes from a library can either be used directly or to make larger membrane structures.
-Examples include $lipidbook$ (\url{lipidbook.bioch.ox.ac.uk}) and $zenodo$ (\url{www.zenodo.org}).
+Examples include $lipidbook$ (\url{https://lipidbook.bioch.ox.ac.uk}) and $zenodo$ (\url{http://www.zenodo.org}).
```
Comment (mimischi) [inline membranes_good.tex]: Lipidbooks seems to be not working anymore? Neither https://lipidbook.bioch.ox.ac.uk nor http://lipidbook.bioch.ox.ac.uk work.

**Comment by mimischi** [comment]
I just finished reading and pushed all commits. If necessary, I can group all/related commits by squashing.

Some more things that came to my eye, but might only be nitpicking:
- A full stop is sometimes added to list items (compare checklist on page 14 and Tables 1-3)
- Whitespace is sometimes added before and after a `/` (forward slash), but omitted other times (i.e. Table 3)
- "X-ray" is written in lowercase ("x-ray") most of the time, but once as "X-ray" (bottom, Table 1). I don't know if there is any "correct" convention.

File: membranes_good.tex (hunk 1 above)
Comment (mimischi) [reply membranes_good.tex]: Paging @orbeckst

Comment (orbeckst) [reply membranes_good.tex]: Thanks, will investigate...

**Review by davesmith4398 — APPROVED** [review]

**Comment by davesmith4398** [comment]
Hi @mimischi - Thank you very much for your edits and suggestions. We have accepted all of your edits. In terms of your additional suggestions:

(1) For consistency, we have added full stops to all list items in the checklist, but not in the tables.
(2) We have eliminated all white space immediately before and after forward slashes.
(3) Per certain bibliographic guidelines, we have eliminated capitalization on the one "X-ray" instance.

@mimischi and @orbeckst - If/when you locate the appropriate lipidbook link, please let us know, and we will make the update. Thank you!

@mrshirts - The appropriate files (including published version 1) should be updated per this conversation.

File: membranes_good.tex (hunk 1 above)
Comment (orbeckst) [reply membranes_good.tex:1091]: https://lipidbook.bioch.ox.ac.uk/ is up again; this is also the correct address

**Comment by mimischi** [comment]
@davesmith4398 Lipidbook is online again, as per https://github.com/davesmith4398/best_practice_membranes/pull/10#discussion_r246194620, and available at https://lipidbook.bioch.ox.ac.uk/.

**Pull request merged**